src/vcpkg/commands.set-installed.cpp

```diff
@@ -109,6 +244,14 @@ namespace vcpkg::Commands::SetInstalled
             }
         }
 
+        if (paths.manifest_mode_enabled() && paths.get_feature_flags().dependency_graph)
+        {
+            auto gh_root_dir = args.github_workspace.value_or_exit(VCPKG_LINE_INFO);
```

Comment:
It doesn't seem like crashing if we don't have a github workspace is an appropriate response?